### PR TITLE
Added Turkish language

### PR DESCRIPTION
**Pull request by B4tuhanY1lmaz**
I've done the translation of website and updated the locale.ts file as shown in the HOWTO.md. I hope it helps to Turkish users of Pacstall.

### CODE REVIEW

File: client/src/locale/locale.ts [hunk 1]
```diff
@@ -165,6 +166,9 @@ export const translations = {
     'id-ID': {
         translation: idLocale,
     },
+    'tr-TR': {
+        translation: trLocale,
+    },
 } as const satisfies Resource
 
 export const localeFlags: Record<keyof typeof translations, string> = {
```
Comment (Henryws) [inline client/src/locale/locale.ts:174]: Check lines 175-185.

**Review by Henryws — CHANGES_REQUESTED** [review]

File: client/src/locale/tr-TR.locale.ts [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    home: {
+        title: 'Pacstall',
+        subtitle: 'Ubuntu için AUR',
+        cards: {
+            whyDifferent: {
+                title: 'Bu diğer paket yöneticilerinden neden farklı?',
+                description:
+                    'Pacstall Ubuntu`nun stabil tabanını ve son sürüm yazılımları +
+                    "hiçbir taviz vermeden kullanmanızı sağlar. Bu şekilde sizi " +
+                    'güvenlik güncellemeleri yada yeni özellikler konusunda büyük +
```
Comment (Henryws) [inline client/src/locale/tr-TR.locale.ts:13]:
```suggestion
                    'güvenlik güncellemeleri yada yeni özellikler konusunda büyük' +
```

File: client/src/locale/tr-TR.locale.ts [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+                    bir endişeden kurtarır ve kolaylık sağlar.',
```
Comment (Henryws) [inline client/src/locale/tr-TR.locale.ts:14]:
```suggestion
                    'bir endişeden kurtarır ve kolaylık sağlar.',
```

**Review by Henryws — CHANGES_REQUESTED** [review]

File: client/src/locale/locale.ts [hunk 4]
```diff
@@ -179,6 +183,7 @@ export const localeFlags: Record<keyof typeof translations, string> = {
     'pt-BR': 'PT 🇧🇷',
     'ro-RO': 'RO 🇷🇴',
     'sv-SE': 'SV 🇸🇪',
+    'tr-TR': 'TR 🇹🇷',
```
Comment (Henryws) [inline client/src/locale/locale.ts:186]: Add this in alphabetical order (exempting English).

**Review by Henryws — CHANGES_REQUESTED** [review]

File: client/src/locale/tr-TR.locale.ts [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+
+export default <Locale>{
+    home: {
+        title: 'Pacstall',
+        subtitle: 'Ubuntu için AUR',
+        cards: {
+            whyDifferent: {
+                title: 'Bu diğer paket yöneticilerinden neden farklı?',
+                description:
+                    'Pacstall Ubuntu`nun stabil tabanını ve son sürüm yazılımları +
```
Comment (Henryws) [inline client/src/locale/tr-TR.locale.ts:11]:
```suggestion
                    'Pacstall Ubuntu`nun stabil tabanını ve son sürüm yazılımları' +
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Comment by B4tuhanY1lmaz** [comment]
Sorry for locale.ts file. Thanks to Lapce ide I wasn't able to see those lines and I trough they were just blank lines but I completed it now.

**Comment by Henryws** [comment]
Can you run:
```bash
npx npx prettier --write client/src/locale/locale.ts client/src/locale/tr-TR.locale.ts
```

**Review by Henryws — APPROVED** [review]

**Comment by Henryws** [comment]
Oops, I forgot to move the list into alphabetical order...

**Comment by Henryws** [comment]
Oh wait it already is lol 😆

**Pull request merged**